### DISCUSSION TITLE
Calling All Developers: Contribute to Bloxtor - A No-Code/Low-Code/Auto-Code Web App Framework

### DISCUSSION THREAD

**Discussion by a19836** [community/community#141539] [Programming Help]
Hello Universe group,

I'm excited to introduce **Bloxtor**, a cutting-edge, professional framework designed to help developers, No-Coders, and Low-Coders alike build powerful web apps. **Bloxtor** is an intelligent framework that transforms code into visual environments and visual components back into code, making development smoother and more intuitive. 

Whether you're just starting out or are a seasoned developer, Bloxtor's versatility provides value for everyone:
- **No-Coders** can visually create apps without diving into complex code.
- **Low-Coders** can enjoy a hybrid approach, using some code alongside the visual builder.
- **Coders** can take advantage of its powerful code parser, seamlessly translating between visual and traditional development.

**And here’s the best part**: Bloxtor is completely **FREE** and **OPEN SOURCE**!

I'm looking for passionate developers who want to contribute to this evolving framework, share their ideas, and help bring Bloxtor to the next level. Whether it's coding, suggesting features, or providing feedback, any contribution will help us grow.

Check out the Bloxtor repository here: [Bloxtor GitHub Repository](https://github.com/a19836/bloxtor)  
Learn more on our homepage: [Bloxtor Homepage](https://bloxtor.com)

Join us in building something innovative and impactful for the developer community!

**What do you think about the Bloxtor framework?** We’d love to hear your thoughts, suggestions, or feedback!

Thank you in advance, and I look forward to your support!

**Comment by ettaboyle** [comment]
Hi @a19836 - Thanks for being a part of the GitHub Community!
- You posted this in the `Universe` category, which is intended for questions about the event, speaker presentation updates, and in general things limited to the actual event. **Are you presenting at Universe - or is there some other tie I'm not aware of?**

It looks like you're trying to drive awareness of your platform, and solicit user feedback. Am I reading this right? **It feels a lot like advertisement.** This could potentially be helpful for users in the `programming help` category, where I can see you already created a post: [Looking for Contributors to Help Improve Bloxtor](https://github.com/orgs/community/discussions/141537#top), but I don't see how this is related to the GitHub event.

- We don't explicitly "ban" advertising in the [GitHub Community Code of Conduct](https://docs.github.com/en/site-policy/github-terms/github-community-code-of-conduct), but its also not really a great fit for this space. Please let me know if I am missing some connection, and share what you're intention is here with this post. 

- If we see more duplicate posts, or any further discussions created by anyone associated with Bloxtor that are also similar to advertising, we'll remove them. 

Thanks for your understanding!

_ps. If we don't hear back from you by Oct. 21st I am going to recategorize and probably close and lock this post. Thanks!_

**Reply by a19836** [reply]
Hello,

Thank you for reaching out and for the clarification.

I apologize for any confusion caused by my post. I now realize that the content I shared may not have been appropriate for the GitHub Universe category. My intention was not to advertise, but rather to gather feedback and contributions from the GitHub community to help improve Bloxtor. 

However, I understand how it may have come across differently. I'll be more mindful of the appropriate categories in the future and will ensure not to post content that could be perceived as promotional in nature in this space.

Regarding your suggestion, I'm happy to stick to the programming help category, where this type of engagement is more fitting. If there are any other guidelines or feedback you'd like to share, I’m open to hearing them to ensure that my participation aligns with the community's values and expectations.

Thanks again for your understanding, and please feel free to recategorize the post as you see fit.

Best regards
